auto3dseg/tasks/hecktor22/hecktor22.py

```diff
@@ -0,0 +1,15 @@
+from monai.apps.auto3dseg import AutoRunner
+
+# the minimum required  code is to create an AutoRunner() and call runner.run()
+# the algos must be set to 'segresnet' (since currently it's the only algo with support of multi-resolution input images, such as CT and PET)
```

Comment:
I think this is an important piece of information. How about making it visible in the README.md file?